iperf/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM clearlinux:latest
+MAINTAINER long1.wang@intel.com
+#RUN swupd mirror -s http://xxx.xxx/pub/mirrors/clearlinux/update
+RUN swupd bundle-add sysadmin-basic
```

Comment:
We should hold off on this change. Better to first create an iperf bundle since this is a small utility and sysadmin-basic is pretty big.

Reply:
create the iperf microservice based on individual iperf bundle.